lab-allie/app/config/router-config.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = [
+  '$stateProvider',
+  '$urlRouterProvider',
+  function($stateProvider, $urlRouterProvider) {
+    // $urlRouterProvider.when('', '/join#signup'); from lecture
+    $urlRouterProvider.when('', '/home'); // from demo code
```

Comment:
I think we talked in class about this after this assignment, but just a reminder that `$urlRouterProvider` is deprecated, and you should not be using `$urlServiceProvider.rules.when()`